File: src/main/java/calebzhou/rdi/microservice/model/json/weather/Precipitation.java

package calebzhou.rdi.microservice.model.json.weather;

import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class Precipitation {

    @SerializedName("local")
    @Expose
    private Local local;

    public Local getLocal() {
        return local;
    }

    public void setLocal(Local local) {
        this.local = local;
    }

}

File: src/main/kotlin/calebzhou/rdi/microservice/model/json/weather/Precipitation.kt
package calebzhou.rdi.microservice.model.json.weather

import com.google.gson.annotations.SerializedName


data class Precipitation (

  @SerializedName("date" ) var date : String? = null,
  @SerializedName("max"  ) var max  : Float?    = null,
  @SerializedName("min"  ) var min  : Double?    = null,
  @SerializedName("avg"  ) var avg  : Double?    = null

)